.github/workflows/release.yml

```diff
@@ -5,12 +5,35 @@ on:
     types: [prereleased, released]
 
 jobs:
+  build:
+    name: Build integration for
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [ linux ]
+        goarch: [ amd64, arm64 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build integration
+        env:
+          GOOS: ${{ matrix.goos }}
+          GOARCH: ${{ matrix.goarch }}
+        run: |
+          make compile
+      - name: Upload artifact for docker build step
+        uses: actions/upload-artifact@v2
```

Comment:
what does this action do? where does it upload to?

Reply:
To a very comfy place where the artifacts can live a peaceful and fulfilling life :)
https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts